driversupport/src/main/java/com/example/upm/androidthings/driversupport/BoardDefaults.java

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-package com.example.upm.androidthings.driverlibrary;
+package com.example.upm.androidthings.driversupport;
 
 import android.content.Context;
 import android.content.res.Resources;
 import android.os.Build;
-import mraa.mraa;
+import mraa.*;
```

Comment:
Why do we need mraa.* ?  BoardDefaults just needs mraa.mraa and not all the classes in mraa are required for this package